app/viewmodels/govuk/ErrorSummaryFluency.scala

```diff
@@ -41,7 +41,9 @@ trait ErrorSummaryFluency {
       errorLinkOverrides: Map[String, String]
     )(implicit messages: Messages): ErrorSummary = {
 
-      val allErrors = errors.distinctBy(_.message).map { error =>
+      val priorityError = errors.headOption.toList
```

Comment:
This is common code. Are you sure this change doesn't break any functionality were we suppose to see multiple errors (e.g. there are multiple fields on the form).

Reply:
Yes, but:

- Is the first one in `error` always the highest priority?
- In my understanding if there are multiple fields on a form and e.g. all have errors all should show one error. I.e. if there are 3 fields, the maximum number of errors shown on the error summary should be 3 (one for each field). I was just highlighting that this could potentially break that. I haven't checked.